pkgs/applications/audio/audacity/default.nix

```diff
@@ -91,6 +92,7 @@ stdenv.mkDerivation rec {
     pkg-config
     python3
     makeWrapper
+    wrapGAppsHook
```

Comment:
Now we double wrap, right?

Reply:
Indeed. Is there a simple way to reduce this on Linux?

Reply:
Disable automatic gapps wrapping and append it to makeWrapper. https://github.com/NixOS/nixpkgs/blob/3a73976c74cac8b4f4c8a061fef5b206d0fd1fea/doc/languages-frameworks/gnome.section.md#when-using-wrapgappshook-with-special-derivers-you-can-end-up-with-double-wrapped-binaries-ssec-gnome-common-issues-double-wrapped

Reply:
I guess my attempt doesn't work according to https://github.com/NixOS/nixpkgs/pull/267963#issuecomment-1815644365, would you please have a look?